src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/RazorConfigurationJsonConverter.cs

```diff
@@ -60,4 +91,50 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             writer.WriteEndObject();
         }
     }
+
+    internal class RazorLanguageVersionObjectJsonConverter : JsonConverter
```

Comment:
private?

Reply:
> Elements defined in a namespace cannot be explicitly declared as private, protected, protected internal or private protected.
[CS1527](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs1527)

Reply:
oo can just have it as a private class